### PR TITLE
chore: Better names for sub-tests

### PR DESCRIPTION
**Pull request by candiduslynx**

### CODE REVIEW

**Review by hermanschaaf — CHANGES_REQUESTED** [review]
table.Name is used in the for loop, it should be part of the name as well so it's possible to differentiate which one failed

File: plugins/source/aws/client/testing.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func AwsCreateMockClient(t *testing.T, ctrl *gomock.Controller, builder func(*te
 
 func validateTagStructure(t *testing.T, tables schema.Tables) {
 	for _, table := range tables.FlattenTables() {
-		t.Run(table.Name, func(t *testing.T) {
+		t.Run("validate tags", func(t *testing.T) {
```
Comment (disq) [inline plugins/source/aws/client/testing.go:93]:
```suggestion
		t.Run("validate tags " + table.Name, func(t *testing.T) {
```
maybe?

Comment (disq) [reply plugins/source/aws/client/testing.go:93]: I was going to use `:` and so on as separator but in the end they all get converted to `_` anyway

Comment (candiduslynx) [reply plugins/source/aws/client/testing.go]: Addressed in 7aaa291ecb0ba282223bee6b8a80066c5279a7b4

File: plugins/source/aws/client/testing.go [hunk 2]
```diff
@@ -119,7 +119,7 @@ func validateMultiplexers(t *testing.T, parentTable *schema.Table) {
 
 func validateSkippedColumns(t *testing.T, tables schema.Tables) {
 	for _, table := range tables.FlattenTables() {
-		t.Run(table.Name, func(t *testing.T) {
+		t.Run(table.Name+"/validate skipped columns", func(t *testing.T) {
```
Comment (hermanschaaf) [inline plugins/source/aws/client/testing.go:122]: the `/` will get converted to an underscore anyway I believe

**Review by hermanschaaf — APPROVED** [review]

File: plugins/source/aws/client/testing.go (hunk 2 above)
Comment (hermanschaaf) [reply plugins/source/aws/client/testing.go:122]: oh ok, good to know

**Pull request merged**